app/views/ControllerInputsView.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * @author Bingzheng Feng
+ * @module views/ControllerInputsView
+ * @requires models/Commands
+ * @requires util/Validator
+ * @requires util/Logger
+ * @requires models/TelemetryData
+ * @requires util/Template
+ * @requires electron
+ * @listens models/TelemetryData~TelemetryData:data_received
+ * @copyright Waterloo Aerial Robotics Group 2016
+ * @licence https://raw.githubusercontent.com/UWARG/WARG-Ground-Station/master/LICENSE
+ * @description Responsible for displaying all the received controller inputs
+ */
+
+var remote = require('electron').remote;
+var Template = require('../util/Template');
+var TelemetryData = remote.require('./app/models/TelemetryData');
+var Logger = remote.require('./app/util/Logger');
+var Validator = require('../util/Validator');
```

Comment:
Remove modules you're not using from here and the file header, like the validator